packages/webapp/src/components/AddNewCrop/index.jsx

```diff
@@ -98,15 +98,41 @@ export default function PureAddNewCrop({
         title={t('CROP.ADD_CROP')}
         value={progress}
       />
-
       <Input
         data-cy="crop-cropName"
         style={{ marginBottom: '40px' }}
         label={t('CROP_CATALOGUE.NEW_CROP_NAME')}
         hookFormRegister={register('crop_common_name', { required: true })}
         errors={getInputErrors(errors, 'crop_common_name')}
       />
-
+      <Input
+        data-cy="crop-cropGenus"
+        style={{ marginBottom: '40px' }}
+        label={t('CROP_CATALOGUE.GENUS')}
+        hookFormRegister={register('crop_genus', {
+          maxLength: { value: 200, message: 'Genus cannot be longer than 200 characters' },
```

Comment:
@wasswarichard 

For some reason I still cannot see your comment did you figure it out?

I will fix this as you said to be a translation message.